organization/egress/app_inputs.py

```diff
@@ -4,24 +4,49 @@
 
 # third-party
 from pydantic import BaseModel, validator
+from pydantic.class_validators import root_validator
 from tcex.input.field_types import Choice, DateTime, always_array, integer, string
 
 
+def validate_tql(cls, values: dict):  # pylint: disable=unused-argument
```

Comment:
This could be a root validator...don't know that that's the right place for it, but I know I've put similar logic in app_inputs.  Do we have a pattern for stuff like this yet?

Reply:
First one I have done... so don't know about a pattern for that.  And yes... its a root validator used farther down:

_tql_none_validation = root_validator(allow_reuse=True)(validate_tql)

Reply:
...I missed that I'd scrolled into a new file.   Looks good to me